src/server/worker/datum/testing.go

```diff
@@ -76,10 +76,12 @@ func (mi *MockIterator) DatumN(index int) []*common.Input {
 		// Warning: this might break some assumptions about the format of this data
 		result = append(result, &common.Input{
 			FileInfo: &pfs.FileInfo{
-				File:      client.NewFile("dummy-repo", "dummy-branch", "dummy-commit", "/path"),
-				FileType:  pfs.FileType_FILE,
-				SizeBytes: uint64(index),
-				Hash:      []byte(fmt.Sprintf("%d", index)),
+				File:     client.NewFile("dummy-repo", "dummy-branch", "dummy-commit", "/path"),
```

Comment:
This file can be deleted. The `MockIterator` isn't used anywhere and doesn't have the same interface as the 2.0 datum iterators.